docs/addons/opa-gatekeeper.md

```diff
@@ -15,181 +15,87 @@ In the context of a Shared Services Platform running on Amazon EKS, platform tea
 - Which OS capabilities a container can execute with.
 - Which times of day the system can be accessed at.
 
-## How does Gatekeeper work with OPA and Kube-mgmt?
+RBAC (role-based access control) can help with some of the scenarios above but **roles are nothing but a group of permissions that you then assign to users leveraging rolebindings.** If for example, a user tries to perform an operation (get, list, watch, create, etc...) that particular user may do so if they have the appropriate role. **Please note that RBAC should be used in conjunction with OPA Gatekeeper policies to fully secure your cluster.**
 
-The Kubernetes API Server is configured to query OPA for admission control decisions when objects (e.g., Pods, Services, etc.) are created, updated, or deleted. The API Server sends the entire Kubernetes object in the webhook request to OPA. OPA evaluates the policies it has loaded using the admission review as input. The diagram below shows the flow between a user making a request to the Kube-API server and how AdmissionReview and AdmissionRequests are made through OPA Gatekeeper.
+## Key Terminology
 
-## Example Policies
-The following policy denies objects that include container images referring to illegal registries:
+- OPA Constraint Framework - Framework that enforces CRD-based policies and allow declaratively configured policies to be reliably shareable
+- Constraint -  A Constraint is a declaration that its author wants a system to meet a given set of requirements. Each Constraint is written with Rego, a declarative query language used by OPA to enumerate instances of data that violate the expected state of the system. All Constraints are evaluated as a logical AND. If one Constraint is not satisfied, then the whole request is rejected.
+- Enforcement Point - Places where constraints can be enforced. Examples are Git hooks, Kubernetes admission controllers, and audit systems.
+- Constraint Template - Templates that allows users to declare new constraints 
+- Target - Represents a coherent set of objects sharing a common identification and/or selection scheme, generic purpose, and can be analyzed in the same validation context
 
-```rego
-package kubernetes.admission
+## Usage
 
-deny[reason] {
-  some container
-  input_containers[container]
-  not startswith(container.image, "hooli.com/")
-  reason := "container image refers to illegal registry (must be hooli.com)"
-}
+```typescript
+import * as ssp from '@aws-quickstart/ssp-amazon-eks';
 
-input_containers[container] {
-  container := input.request.object.spec.containers[_]
-}
+const app = new cdk.App();
+const account = <AWS_ACCOUNT_ID>;
+const region = <AWS_REGION>;
+const env: { account, region },
```

Comment:
you don't need env if it is only for account/region. You set those on lines 39 and 40.

Reply:
addressed